iceberg/pom.xml

```diff
@@ -26,7 +26,7 @@
     <hive.path.to.root>..</hive.path.to.root>
     <path.to.iceberg.root>.</path.to.iceberg.root>
     <iceberg.version>1.3.0</iceberg.version>
-    <kryo-shaded.version>4.0.3</kryo-shaded.version>
+    <kryo.version>5.0.3</kryo.version>
```

Comment:
@devaspatikrishnatri I see in the attached dependency tree the kryo version is 5.2.0 `com.esotericsoftware:kryo:jar:5.2.0:compile`
While here i see its 5.0.3, is this property not taking effect?